CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ A breaking change will get clearly marked in this log.
 
 ## Unreleased
 
+Added:
+
+- Add stateChanges in SimulateTransaction API response ([#963](https://github.com/stellar/js-stellar-sdk/pull/963))
```

Comment:
This should have a bit more detail, e.g. at least the schema change and mention `rpc.Server.simulateTransaction`